go.mod

```diff
@@ -96,6 +96,7 @@ require (
 	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
 	github.com/pkg/browser v0.0.0-20170505125900-c90ca0c84f15
 	github.com/pkg/errors v0.8.1
+	github.com/prometheus/common v0.0.0-20181126121408-4724e9255275
```

Comment:
where is this coming from?

Reply:
probably the switch from using the sonaboy dynamic client to the kubernetes one, I can check

Reply:
just checked and this can be removed, pushing that now